src/components/PeopplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,138 @@
+import { useState, useEffect } from 'react';
+import classNames from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const { slug: selectedPersonSlug } = useParams();
+
+  useEffect(() => {
+    const getPeopleFromServer = async () => {
+      const peopleFromServer = await getPeople();
+
+      const peopleWithParents = peopleFromServer.map(person => {
+        const newPerson = { ...person };
+
+        const mother = peopleFromServer.find(human => (
+          human.name === person.motherName));
+
+        const father = peopleFromServer.find(human => (
+          human.name === person.fatherName
+        ));
+
+        if (mother) {
+          newPerson.mother = mother;
+        }
+
+        if (father) {
+          newPerson.father = father;
+        }
+
+        return newPerson;
+      });
+
+      setPeople(peopleWithParents);
+    };
+
+    getPeopleFromServer();
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      {people.length
```

Comment:
According to the task checklist table below should be in a separate component.
`8. Pass people to PeopleTable component and render as a table (see the given markup);`